picard/ui/options/interface.py

```diff
@@ -169,6 +168,12 @@ def __init__(self, parent=None):
         super().__init__(parent)
         self.ui = Ui_InterfaceOptionsPage()
         self.ui.setupUi(self)
+
+        self.ui.ui_theme.clear()
+        for theme in ["System", "Light", "Dark"] if IS_WIN or IS_MACOS else ["System"]:
+            self.ui.ui_theme.addItem(theme)
```

Comment:
This currently breaks the theme options on Linux. With this change the only option on Linux is using the Qt theme configured on the system, something we explicitly did not want to have. Instead the default should be that the "Fusion" theme is used and the user can opt-in to use the system theme. For that I think it would make sense to rename the current "System" to "Default" and have a separate "System theme" option for Linux.